src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -657,6 +663,11 @@ public bool ParameterCanBeIdentifier(Features features, int index)
                 paramIdentifierStatus == ParamIdentifierStatus.PossiblyIdentifier;
         }
 
+        public virtual bool ParameterCanBeIdentifier(TexlNode node, int index, Features features)

```

Comment:
>ParameterCanBeIdentifier [](http://example.com/codeflow?start=28&length=24)

Please add a comment explaining why this is needed instead of just using the original one

Reply:
I went ahead and removed this overload. I make changes to PA if necessary.